### DISCUSSION TITLE
Theme system

### DISCUSSION THREAD

**Discussion by Chewbaka69** [lanedirt/OGameX#114] [Ideas]
Maybe theme system, that allow user to change and allow to dev to re-create and use the old ogame design

**Comment by lanedirt** [comment]
Thanks for your suggestion!

Right now the main focus is to build the game functionality according to how it works and what it looks like in the redesign version. However I agree, it should technically be possible to swap out the theme files and recreate the old ogame design feel while keeping the backend logic as-is.

Interesting idea. I might be able to try and play around with this idea to see if I can get a simple prototype working where the template can be switched.

**Reply by Chewbaka69** [reply]
Oh yes totally agree this is not currently the first thing to do :)
